data/map.txt

```diff
@@ -30694,3 +30700,2834 @@ planet Zug
+	government "Gegno Vi"
+	"required reputation" 1
+	bribe 0
+	security 0
+
+planet Essime
+	landscape land/mountain4
+	description "Starting from an early age, most Vi devote their lives to their endless trials of survival, finding themselves either on the frontlines of conflict or in the backlines of industry. As they grow old, they retire and spend the rest of their lives on green planets such as this one, where they hunt animals for food and till acres of land. "
+	spaceport "The spaceport here is a very large circular shape with tall pylons reaching up into a dome roofing. The empty space between each pylon is occasionally interrupted by a freighter or two, and in the center of it is a main center shaft where most pilots are gathered. Surrounding the pilots are what appear to be older members of the Vi deep in conversation."
+	spaceport "	Several Vi are chuckling deeply, likely telling stories of hunts they've had on the planet. Strangely, they pay you no attention; not even of distrust, and carry on about their day as if you aren't even present."
+	government "Gegno Vi"
+	"required reputation" 1
+	bribe 0
+	security 0
+
+planet "Sies Upi"
+	landscape land/hills1
+	description "Sies Upi is one of the most recently colonized planets of the Gegno Vi. This hot, dry world with harsh living environments serves as more of an outpost than it does a colony, but nevertheless the Vi found ways to make manageable living conditions."
+	description "	Being the farthest away from the rest of Gegno territory, this planet is also the first line of defense between the Gegno and the Quarg. Although there are no active hostilities between the two, the Vi remain alert and ready at all times."
+	spaceport "This spaceport seems a lot more militarized than other Gegno planet spaceports, full of geared up Vi with their ships parked close by. The roar of an enormous cruiser's engines can be heard just as often as the shutting down of another. There doesn't look to be very many non-personnel civilians here, and it feels somewhat claustrophobic with all of the guards standing around."
+	government "Gegno Vi"
+
+planet Siedi
+	landscape land/forest5
+	description "The gravitational pull of the nearby gas giant resulted in the large, bulky, mountainous surface of this planet. Tectonic shifts are very common, and the surface is marked with the wear and tear of massive tremors. Beneath the cracks, however, lies a rich source of materials that the Vi quickly capitalized on for their nearby worlds."
+	spaceport "The spaceport is haphazardly build inside the side of a smaller mountain, and every now and then you can feel a very faint rumbling from below. You're not sure if it is beneath you or somewhere off in the distance. The area around the spaceport continues to be under construction, but it is more likely mining for materials than improvements to the structure itself."
```

Comment:
```suggestion
	spaceport "The spaceport is haphazardly build inside the side of a smaller mountain, and every now and then you can feel a very faint rumbling from below. You're not sure if it is beneath you or somewhere off in the distance. The area around the spaceport continues to be under construction, but it is more likely because the Vi are mining for materials here rather than making improvements to the structure itself."
```